product_configurator_wizard/wizard/product_configurator.py

```diff
@@ -783,7 +786,8 @@ def action_config_done(self):
         if self.product_id:
             self.product_id.write({
                 'attribute_value_ids': [(6, 0, self.value_ids.ids)],
-                'value_custom_ids': [(6, 0, custom_vals)]
+                'value_custom_ids': map(lambda cv: (2, cv), self.product_id.value_custom_ids.ids) + \
+                                        self.product_id.product_tmpl_id.encode_custom_values(custom_vals),
```

Comment:
Not sure what would be the effect here, the idea behind using 6, 0 a.k.a (delete all and add the following) is to prevent handling of all possible cases, which can even occur all at once (Link an item, edit an item, delete an item, create and link an item)

If only remove (2) is used I think some of these cases might fail